kpi/management/commands/sync_kobocat_xforms.py

```diff
@@ -367,8 +367,7 @@ def _sync_permissions(asset, xform):
     current_kpi_perms = defaultdict(set)
     for user, kpi_permission in ObjectPermission.objects.filter(
                 deny=False,
-                content_type=ASSET_CT,
-                object_id=asset.pk
+                asset=asset,
```

Comment:
Same question? 
Should it be better to use asset's PK directly. e.g. `asset_id=asset.pk`

Reply:
https://github.com/kobotoolbox/kpi/pull/2580#discussion_r567208901